user/serializers.py

```diff
@@ -1 +1,35 @@
 # write your code here
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+
+
+class UserSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = get_user_model()
+        fields = (
+            "id",
+            "username",
+            "email",
+            "password",
+            "is_staff",
+        )
+        read_only = (
+            "id",
+            "is_staff",
+        )
```

Comment:
The `read_only` attribute in the `Meta` class should be `read_only_fields`. This is the correct attribute name for specifying fields that should be read-only in Django REST Framework serializers.